.env.example

```diff
@@ -10,6 +10,12 @@ REDIS_DSN=redis://localhost
 OSU_API_KEY=
 
 DOMAIN=cmyui.xyz
+
+# Here is how mirror type work [also official mirror url]
+# 1 = Chimu api, https://api.chimu.moe/v1
+# 1 = Cheesegull api, https://storage.ripple.moe/
+# 3 = Nerina api, https://api.nerina.pw/
+MIRROR_TYPE=1
```

Comment:
i wonder if it's possible to use `MIRROR_TYPE(str, Enum)` so we can have this as a string rather than an int?

Reply:
> i wonder if it's possible to use `MIRROR_TYPE(str, Enum)` so we can have this as a string rather than an int?

I think it work? I use number because it easy for user settings

I can change it to string if your want

Reply:
yeah i'd prefer it as a string just for readability - thanks!